bom/application/pom.xml

```diff
@@ -353,6 +353,15 @@
                 <scope>import</scope>
             </dependency>
 
+            <!-- Oracle JDBC driver -->
+            <dependency>
+                <groupId>com.oracle.database.jdbc</groupId>
+                <artifactId>ojdbc-bom</artifactId>
+                <version>21.3.0.0</version>
```

Comment:
Shouldn't you use the `${oracle-jdbc.version}` instead and update the property?

Reply:
right, sure

Reply:
done